src/components/PersistentDrawerLeft/PersistentDrawerLeft.js

```diff
@@ -64,6 +64,8 @@ const useStyles = makeStyles((theme) => ({
   drawerPaper: {
     width: drawerWidth,
     // overflow: "hidden",
+     backgroundColor:"#1e1e1e",
```

Comment:
what is the need of these

Reply:
I don't think I have done anything here....

Reply:
I have tested it I dont think I found a bug